pyteal/ast/base64decode_test.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+
+import pyteal as pt
+
+teal6Options = pt.CompileOptions(version=6)
+teal7Options = pt.CompileOptions(version=7)
+
+
+def test_base64decode_std():
+    arg = pt.Bytes("aGVsbG8gd29ybGQ=")
+    expr = pt.Base64Decode.std(arg)
+    assert expr.type_of() == pt.TealType.bytes
+
+    expected = pt.TealSimpleBlock(
+        [
+            pt.TealOp(arg, pt.Op.byte, '"aGVsbG8gd29ybGQ="'),
+            pt.TealOp(expr, pt.Op.base64_decode, "StdEncoding"),
+        ]
+    )
+
+    actual, _ = expr.__teal__(teal7Options)
+    actual.addIncoming()
+    actual = pt.TealBlock.NormalizeBlocks(actual)
+
+    with pt.TealComponent.Context.ignoreExprEquality():
+        assert actual == expected
```

Comment:
@jdtzmn Minor and optional:  The _expr_ values are equal here.  Optionally, I'd consider enforcing complete equality.

If you'd prefer as is, feel welcomed to resolve the thread.

```suggestion
    assert actual == expected
```